packages/synapse-react-client/src/components/DataGrid/utils/extractColumnValidationMessages.ts

```diff
@@ -4,12 +4,15 @@
  * Example:
  *  extractColumnValidationMessages(["#/platform: null is not a valid enum value"])
  *  -> Map { "platform" => ["null is not a valid enum value"] }
+ *
+ *  extractColumnValidationMessages(["#/requiredNumberArrayColumn/0: some error"])
+ *  -> Map { "requiredNumberArrayColumn" => ["some error"] }
  */
 export function extractColumnValidationMessages(
   messages: string[],
 ): Map<string, string[]> {
   const columnMap = new Map<string, string[]>()
-  const regex = /^#\/([^:]+):\s*(.*)$/
+  const regex = /^#\/([^/:]+)(?:\/\d+)?:\s*(.*)$/
```

Comment:
Regex needed to be updated for list columns that provide validation messages per-element, like `#/column/0: expected string, got null`